tests/macos/xcodeproj/Single-Application-Project-AllTargets.xcodeproj/project.pbxproj

```diff
@@ -415,11 +420,12 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
 				FRAMEWORK_SEARCH_PATHS = "";
-				GCC_PREPROCESSOR_DEFINITIONS = "\"REQUIRED_DEFINED_FLAG\"";
+				GCC_PREPROCESSOR_DEFINITIONS = "\"REQUIRED_DEFINED_FLAG=1\" \"FLAG_WITH_VALUE_ZERO=0\" $(inherited)";
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				MACH_O_TYPE = "$(inherited)";
 				PRODUCT_NAME = "Single-Application-UnitTests";
 				SUPPORTS_MACCATALYST = 0;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "\"$(inherited)\" \"REQUIRED_DEFINED_FLAG=1\" \"REQUIRED_DEFINED_FLAG\" \"FLAG_WITH_VALUE_ZERO=0\"";
```

Comment:
Over here, `FLAG_WITH_VALUE_ZERO` should not be present, and `REQUIRED_DEFINED_FLAG` should only be present once. It should look like `REQUIRED_DEFINED_FLAG`